assets/airports/kpvd.json

```diff
@@ -0,0 +1,6570 @@
+                "16": "",
+                "34": ""
+            },
+            "body": ["PUT", "NELIE"],
+            "exitPoints": {
+                "BAF": "BAF"
+            },
+            "draw": [["PUT", "NELIE", "BAF"]]
+        },
+        "JUMPR": {
+            "icao": "JUMPR",
+            "name": "Jumper",
+            "rwy": {
+                "05": "",
+                "23": "",
+                "16": "",
+                "34": ""
+            },
+            "body": ["NICOD", "FALMA"],
+            "exitPoints": {
+                "JUMPR": ["JUMPR"]
+            },
+            "draw": [["NICOD", "FALMA", "JUMPR"]]
+        }
+    },
+    "stars": {
+        "JORDN2": {
+            "icao": "JORDN2",
+            "name": "Jordan Two",
+            "entryPoints": {
+                "HTO": ["HTO", "JORDN"]
+            },
+            "body": ["MINNK"],
+            "draw": [["HTO*", "JORDN", "MINNK"]]
+        },
+        "WIPOR3": {
+            "icao": "WIPOR3",
+            "name": "Wipor Three",
+            "entryPoints": {
+                "ALB": ["ALB", ["SHIGY", "A180"]],
+                "HNK": ["HNK", ["HINGZ", "A290"]],
+                "HINGZ": [["HINGZ", "A290"]]
+            },
+            "body": ["PONEE", ["NELIE", "A210"], ["WIPOR", "A110"], "ORW", "SAUGY"],
+            "rwy": {
+                "05": ["LAFAY"],
+                "34": ["LAFAY"],
+                "16": ["WAWIK", "SEABE", "NOSUH"],
+                "23": ["WAWIK", "SEABE", "NOSUH"]
+            },
+            "draw": [
+                ["ALB*", "SHIGY", "PONEE"],
+                ["HNK*", "HINGZ*", "PONEE"],
+                ["PONEE", "NELIE", "WIPOR", "ORW", "SAUGY"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "KPVD",
+            "destination": "",
+            "category": "departure",
+            "route": "KPVD.BAF.BAF",
+            "method": "cyclic",
+            "altitude": null,
+            "speed": null,
+            "rate": 2,
+            "airlines": [
+                ["aal",   7],
+                ["cfg",   1],
+                ["dal",   4],
+                ["jbu",   2],
+                ["swa",   8],
+                ["ual",   4]
+            ]
+        },
+        {
+            "origin": "KPVD",
+            "destination": "",
+            "category": "departure",
+            "route": "KPVD.JUMPR.JUMPR",
+            "method": "cyclic",
+            "altitude": null,
+            "speed": null,
+            "rate": 2,
+            "airlines": [
+                ["aal",   7],
+                ["cfg",   1],
+                ["dal",   4],
+                ["jbu",   2],
+                ["swa",   8],
+                ["ual",   4]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KPVD",
+            "category": "arrival",
+            "route": "HTO.JORDN2.KPVD",
+            "method": "cyclic",
+            "altitude": [12000, 10000],
```

Comment:
I'm not 100% sure if it might be okay, but it may easily be the case that we are choosing an altitude "greater than or equal to 12000 AND less than or equal to 1000", which has no solution. Just to be safe, please change this and others to `[minimum, maximum]`.